sale_invoiced_state/models/sale_order.py

```diff
@@ -0,0 +1,90 @@
+# Copyright 2022 PyTech SRL - Alessandro Uffreduzzi
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+from odoo.tools import float_is_zero, float_compare
+
+
+class SaleOrder(models.Model):
+    _inherit = "sale.order"
+
+    invoiced_state = fields.Selection(
+        [
+            ("no", "No"),
```

Comment:
My main reasoning was that "no" is pretty unambiguous: There have been no invoices. Whereas "yes" is more ambiguous, since it can be invoiced but only partially. I don't mind changing it, though.